contribution_description.md

```diff
@@ -0,0 +1,14 @@
+
+# freeCodeCamp
```

Comment:
move to `projects/javascript/freecodecamp.md`

Reply:
moved to appropriate location